environment.yml

```diff
@@ -4,10 +4,11 @@ channels:
 - conda-forge
 - default
 dependencies:
-- biorbd >=1.7.2
+- biorbd >=1.8.3
```

Comment:
Explicitely expression biorbd is probably not required anymore (that was a hack for a bug in conda which is now fixed)

Reply:
Meaning this line can be removed ?